client/common_test.go

```diff
@@ -140,6 +140,44 @@ var _ = Describe("common test", func() {
 			Expect(err).To(HaveOccurred())
 		})
 
+		It("with Language", func() {
+			request, _ := http.NewRequest("GET", fmt.Sprintf("%s%s", jenkinsCore.URL, "/view/all/itemCategories?depth=3"), nil)
+			response := &http.Response{
+				StatusCode: 200,
+				Proto:      "HTTP/1.1",
+				Request:    request,
+				Body: ioutil.NopCloser(bytes.NewBufferString(`number name                       type
+0      构建一个自由风格的软件项目 Standalone Projects
+1      构建一个maven项目          Standalone Projects
+2      流水线                     Standalone Projects
+3      构建一个多配置项目         Standalone Projects
+0      Bitbucket Team/Project     Nested Projects
+1      文件夹                     Nested Projects
+2      GitHub 组织                Nested Projects
+3      多分支流水线               Nested Projects
+`)),
+			}
+			request.Header.Set("Accept-Language", "zh-CN")
+			roundTripper.EXPECT().
+				RoundTrip(request).Return(response, nil)
+
+			language = "zh-CN"
```

Comment:
Please use `SetLanguage("zh-CN")` instead of `language = "zh-CN"`

Reply:
Yeah. I think so.